lib/nerves_time/rtc/abracon/b5ze/date.ex

```diff
@@ -18,15 +18,14 @@ defmodule NervesTime.RTC.Abracon.B5ZE.Date do
   """
   @spec decode(<<_::56>>) :: {:ok, NaiveDateTime.t()} | {:error, any()}
   def decode(<<seconds_bcd, minutes_bcd, hours24_bcd, day_bcd, _weekday, month_bcd, year_bcd>>) do
-    {:ok,
```

Comment:
The `{:ok, ` tuple part got dropped. I'm surprised this didn't break a test...

Reply:
Wow, Dialyzer didn't catch it either...

Reply:
That's because `NaiveDateTime.new/8` returns `{:ok, t()} | {:error, atom()}` - So its the same

https://hexdocs.pm/elixir/NaiveDateTime.html#new/8